### PR TITLE
New additions to pipeline guidelines

### PR DESCRIPTION
**Pull request by ewels**
@netlify /docs/guidelines/pipelines/recommendations/build_with_community

### CODE REVIEW

File: src/content/docs/guidelines/pipelines/requirements/nextflow.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Nextflow
+subtitle: Pipelines must be built with Nextflow
+menu:
+  main:
+    weight: 3
+---
+
+Pipelines must be built with Nextflow.
```
Comment (jfy133) [inline src/content/docs/guidelines/pipelines/requirements/nextflow.md:9]: But about those poor python developers who can't write in anything else?!

**Review by jfy133 — APPROVED** [review]

**Review by FriederikeHanssen — COMMENTED** [review]
Sounds good to me. Maybe we can link from the Tutorial > add a new pipeline to the guidelines or also add a sentence there. It was the first place I checked when opening the website in previewimage

**Comment by christopher-hakkaart** [comment]
Looks good to me also

**Comment by ewels** [comment]
Agree that we should link to the guidelines more prominently from the tutorial 👍🏻 (there is a link there but it's a bit hidden).

Can take that in a separate PR though if that's ok.

**Pull request merged**